spec/rack/test/cookie_spec.rb

```diff
@@ -216,4 +216,10 @@
       last_request.cookies.should == { "value" => "1" }
     end
   end
+
+  it "allow symbol access" do
+    jar = Rack::Test::CookieJar.new
+    jar["value"] = "foo;abc"
+    jar[:value].should == "foo;abc"
+  end
```

Comment:
@Anorlondo448 are you still available for the change that has been requested.
Otherwise I would just "re-implement" the change on the current master and open a PR that places the test at the right spot. What do you think @perlun ?